app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Users::RegistrationsController < Devise::RegistrationsController
+    # Override the action you want here.
+  before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
```suggestion
  before_action :configure_permitted_parameters
```
we are already in devise controller